client/src/locales/en/messages.po

```diff
@@ -858,11 +866,15 @@ msgstr "The city borough where your search address is located"
 msgid "The city’s fifth <0>worst evictor</0> in 2018, A&E is a prime example of a landlord who engages in <1>aggressive eviction strategies</1> to displace low-income tenants. Besides lack of repairs and frivolous evictions in housing court, A&E has also been known to use <2>MCIs</2> to <3>double rents</3> in rent-stabilized buildings."
 msgstr "The city’s fifth <0>worst evictor</0> in 2018, A&E is a prime example of a landlord who engages in <1>aggressive eviction strategies</1> to displace low-income tenants. Besides lack of repairs and frivolous evictions in housing court, A&E has also been known to use <2>MCIs</2> to <3>double rents</3> in rent-stabilized buildings."
 
-#: src/components/PropertiesSummary.tsx:108
+#: src/components/PropertiesSummary.tsx:109
 msgid "The most common corporate entity is <0>{0}</0> and the most common business address is <1>{1}</1>."
 msgstr "The most common corporate entity is <0>{0}</0> and the most common business address is <1>{1}</1>."
 
-#: src/components/PropertiesSummary.tsx:102
+#: src/components/ComplaintsSummary.tsx:18
+msgid "The most common reported in this portfolio over the last three years <0/>."
```

Comment:
does this just exclude the "issue" or "issues" because of the plural thing?

Reply:
I believe so! it's a little odd tho, lemme look in to how other Plural tags show up in these files.

Reply:
omg!! @samaratrilling this is so annoying but the reason these messages werent being extracted properly was because of a very bizarre error where if you import the Plural component _before_ you import the Trans component lingui cannot recognize the Plural tags in strings.... I fixed it in this PR but gonna flag it for review tomorrow as somethign we should brainstorm about documenting bc it's super annoying and not intuitive...

Reply:
We can definitely write a custom prettier rule for this!